routesrv/routesrv.go

```diff
@@ -91,6 +84,26 @@ func New(opts Options) (*RouteServer, error) {
 		oauthConfig.CallbackPath = opts.OAuth2CallbackPath
 	}
 
+	var rh *RedisHandler
+	// in case we have kubernetes dataclient and we can detect redis instances, we patch redisOptions
+	if opts.KubernetesRedisServiceNamespace != "" && opts.KubernetesRedisServiceName != "" {
+		log.Infof("Use endpoints %s/%s to fetch updated redis shards", opts.KubernetesRedisServiceNamespace, opts.KubernetesRedisServiceName)
+		rh = &RedisHandler{}
+		_, err := dataclient.LoadAll()
+		if err != nil {
+			return nil, err
+		}
+		rh.AddrUpdater = getRedisAddresses(opts.KubernetesRedisServiceNamespace, opts.KubernetesRedisServiceName, dataclient)
+		handler.Handle("/swarm", rh)
```

Comment:
@AlexanderYastrebov what do you think about /swarm as path endpoint?

Reply:
I would call it redis, not sure we need to abstract here

Reply:
maybe /swarm/redis/endpoints or /swarm/redis/shards ?
Like this we can add `/swarm/swim/*` later